pycobertura/reporters.py

```diff
@@ -38,33 +39,40 @@ def get_report_lines(self):
                 for filename in self.cobertura.files()
             ],
             "Cover": [
-                self.format_line_rate(self.cobertura.line_rate(filename))
-                for filename in self.cobertura.files()
+                self.format_line_rate(self.cobertura.line_rate(filename)) for filename in self.cobertura.files()
             ],
             "Missing": [
                 stringify(self.cobertura.missed_lines(filename))
                 for filename in self.cobertura.files()
             ],
         }
-        lines["Filename"].append("TOTAL")
-        lines["Stmts"] += [self.cobertura.total_statements()]
-        lines["Miss"] += [self.cobertura.total_misses()]
-        lines["Cover"] += [self.format_line_rate(self.cobertura.line_rate())]
-        lines["Missing"].append("")
+        footer_func_dict = {
+            "Stmts": [self.cobertura.total_statements()],
+            "Miss": [self.cobertura.total_misses()],
+            "Cover": [self.format_line_rate(self.cobertura.line_rate())],
+            "Missing": [""],
+        }
+        return rows_func_dict[key] + footer_func_dict[key]
+
+    def get_report_lines(self):
```

Comment:
I'd like to rename this to `get_summary_lines()` (and elsewhere when it applies), which is more self-explanatory. The wording "report" in this context is too generic because reports include the summary and source code lines. If you agree, I think we should also make other variables such as "lines" be renamed to "summary_lines", whenever that context applies. What do you think?